pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -127,6 +127,12 @@ func (pas *PodAutoscalerStatus) IsReady() bool {
 	return podCondSet.Manage(pas.duck()).IsHappy()
 }
 
+// IsUnready returns true if the Ready condition is false.
+func (pas *PodAutoscalerStatus) IsUnready() bool {
```

Comment:
Why not just use !IsHappy()?

Reply:
Because Ready has 3 states, !IsReady() != IsUnready().